rfc/draft-oberstet-hybi-tavendo-wamp-00.md

```diff
@@ -1138,7 +1131,7 @@ Upon subscribing to a topic via the `SUBSCRIBE` message, a *Subscriber* will rec
 
 A subscription lasts for the duration of a session, unless a *Subscriber* opts out from a previously established subscription via the `UNSUBSCRIBE` message.
 
-> A *Subscriber* may have more than one event handler attached to the same subscription. This can be implemented in different ways: a) a *Subscriber* can recognize itself that it is already subscribed and just attach another handler to the subscription for incoming events, b) or it can send a new `SUBSCRIBE` message to broker (as it would be first) and upon receiving a `SUBSCRIBED.Subscription|id` it already knows about, attach the handler to the existing subscription
+> A *Subscriber* may have more than one event handler attached to the same subscription. This can be implemented in different ways: a) a *Subscriber* can recognize itself that it is already subscribed and just attach another handler to the subscription for incoming events, b) or it can send a new `SUBSCRIBE` message to broker (as it would be first) and upon receiving a `SUBSCRIBED.Subscription|id` it already knows about, attach the handler to the existing subscription
```

Comment:
I don't know why it thinks I changed this line (and all the following ones). It seems like only the whitespace changed. I can undo this (and the following diffs) in a subsequent commit.